deps/chakrashim/src/jsrtutils.cc

```diff
@@ -1018,18 +961,23 @@ StringUtf8::~StringUtf8() {
 
 JsErrorCode StringUtf8::From(JsValueRef strRef) {
   CHAKRA_ASSERT(!_str);
+  
   size_t len = 0;
   IfJsErrorRet(JsCopyString(strRef, nullptr, 0, &len));
+  
   char* buffer = reinterpret_cast<char*>(malloc(len+1));
   CHAKRA_VERIFY(buffer != nullptr);
+  
   size_t written = 0;
   JsErrorCode errorCode = JsCopyString(strRef, buffer, len, &written);
+  
   if (errorCode == JsNoError) {
     CHAKRA_ASSERT(len == written);
     buffer[len] = '\0';
     _str = buffer;
     _length = static_cast<int>(len);
   }
+  
```

Comment:
extra spaces? can you run cpplint on all changed files and make sure you didn't introduced new error/warnings